thirdpartynotices.html

```diff
@@ -18,4 +484,511 @@
+<li>Copyright (c) 1995-2017 Jean-loup Gailly and Mark Adler</li>
+<li>Copyright (c) 2015 Xamarin, Inc (http://www.xamarin.com)</li>
+<li>Copyright (c) 2009, 2010, 2013-2016 by the Brotli Authors.</li>
+<li>Copyright (c) YEAR W3C(r) (MIT, ERCIM, Keio, Beihang). Disclaimers THIS WORK IS PROVIDED AS</li></ul>
+        <pre>
+        The MIT License (MIT)
+
+Copyright (c) .NET Foundation and Contributors
+
+All rights reserved.
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the &quot;Software&quot;), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED &quot;AS IS&quot;, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+
+        </pre>
+    </details>
+</li>
+<li>
+    <details>
+        <summary>
+            System.Threading.Tasks.Extensions 4.5.1 - MIT
+        </summary>
+        
+        
+        <pre>
+        The MIT License (MIT)
+
+Copyright (c) .NET Foundation and Contributors
+
+All rights reserved.
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the &quot;Software&quot;), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED &quot;AS IS&quot;, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+
+        </pre>
+    </details>
+</li>
+<li>
+    <details>
+        <summary>
+            WiX 3.11.2 - MS-RL
```

Comment:
I'm not sure this is accurate; we distribute an MSI built with WiX, do we really distribute WiX itself? If we do, I think the MS-RL demands certain copyleft obligations our release process probably isn't including right now

Reply:
(if it's not accurate, that probably suggests that you need to go disable this package from the "Configure NOTICE generation" UI in component governance, then regenerate the file)

Reply:
We do not distribute WiX separately from the MSI we build with it. Our current [ai-win thirdpartynotices file includes the same notice](https://github.com/microsoft/accessibility-insights-windows/blob/e896138d51854a1eb928e6aa3a509791dd3e7a8b/thirdpartynotices.html#L669). @DaveTryon, do you have any background on the inclusion of the WiX notice?

Reply:
@jalkire, @dbjorge, our last guidance from legal was that we didn't need the MS-RL license, but that we had an "alternative license" for WiX. I'll forward you both the email but I don't have specifics about what "alternative license" means here...

Reply:
Removed wix & regenerated file per email guidance